parrot_ai/prompts/sermon.py

```
@@ -224,55 +214,58 @@
-* Structure_Cohesion: 0.15
-* Illustrations: 0.10
-* FCF_Identification: 0.10
-
-Formula: sum(weight_i * score_i)."""
\ No newline at end of file
+* Proposition_Clarity ≈ avg(Proposition.Principle_and_Application_Wed, Proposition.Establishes_Main_Theme, Proposition.Summarizes_Introduction)
+* FCF_Identification ≈ Introduction.FCF_Introduced
+* Application_Effectiveness ≈ avg(Application.Clear_and_Practical, Application.Redemptive_Focus, Application.Mandate_vs_Idea_Distinction, Application.Passage_Supported, Main_Points.Application_Quality)
+* Structure_Cohesion ≈ avg(Main_Points.Proportional_and_Coexistent, Conclusion.Summary, Conclusion.Compelling_Exhortation, Conclusion.Climax, Conclusion.Pointed_End)
+* Illustrations ≈ avg(Main_Points.Illustration_Quality, Illustrations.Lived_Body_Detail, Illustrations.Strengthens_Points, Illustrations.Proportion)
+* Overall_Impact ≈ avg(Textual_Fidelity, Proposition_Clarity, Application_Effectiveness, Structure_Cohesion, Illustrations, FCF_Identification)"""
```

Ruff (0.13.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)